release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2396,6 +2396,116 @@ $ ./openshift-install wait-for install-complete
+As a workaround, include a status check in the catalog source policy in the `spec` section.
++
+[source,yaml]
+----
+metadata:
+  name: redhat-operators-disconnected
+spec:
+  displayName: disconnected-redhat-operators
+  image: registry.example.com:5000/disconnected-redhat-operators/disconnected-redhat-operator-index:v4.11
+status:
+  connectionState:
+    lastObservedState: READY
+----
++
+This mitigates the delay for OLM to pull the new index image and get the pod ready, reducing the overall time between completion of catalog source policy remediation and the update of the subscription status. 
+If the issue persists and the subscription policy status update is still late with the workaround, apply an identical `ClusterGroupUpdate` CR with a different name. 
+(link:https://issues.redhat.com/browse/OCPBUGS-2813[*OCPBUGS-2813*])
+
+* {cgu-operator} skips remediating a policy if all selected clusters are compliant when the `ClusterGroupUpdate` CR is started.
+The update of operators with a modified catalog source policy and a subscription policy in the same `ClusterGroupUpdate` CR does not complete. 
+The subscription policy is skipped as it is still compliant until the catalog source change is enforced.
+As a workaround, add a trivial change to one CR in the `common-subscription` policy, for example: 
++
+[source,yaml]
+----
+metadata.annotations.upgrade: "1"
+----
++
+This makes the policy non-compliant prior to the start of the `ClusterGroupUpdate` CR. 
+(link:https://issues.redhat.com/browse/OCPBUGS-2812[*OCPBUGS-2812*])
+
+* On a {sno} instance, rebooting without draining the node to remove all the running pods can cause issues with workload container recovery. 
+After the reboot, the workload restarts before all the device plugins are ready, resulting in resources not being available or the workload running on the wrong NUMA node.
+The workaround is to restart the workload pods when all the device plugins have re-registered themselves during the reboot recovery procedure. 
+(link:https://issues.redhat.com/browse/OCPBUGS-2180[*OCPBUGS-2180*])
+
+* The default `dataset_comparison` is currently `ieee1588`. The recommended dataset_comparison is `G.8275.x`. Work on correcting this in progress. In the short term, you can manually update the ptp configuration to include the recommended `dataset_comparison`. 
+(link:https://issues.redhat.com/browse/OCPBUGS-2336[*OCPBUGS-2336*])
+
+* The default `step_threshold` is 0.0. The recommended `step_threshold` is 2.0. Work on correcting this is in progress. In the short term, you can manually update the ptp configuration to include the recommended `step_threshold`. 
+(link:https://issues.redhat.com/browse/OCPBUGS-3005[*OCPBUGS-3005*])
+
+* The `BMCEventSubscription` CR fails to create a Redfish subscription in an ACM-deployed multi-cluster environment, where the metal3 service is only running on a hub cluster.
```

Comment:
Suggest to add a few words:

`fails to create a Redfish subscription for a spoke cluster in an ACM-deployed`

Reply:
added.
Thank you